catalog/spatial/spatial-app/src/main/resources/features.xml

```diff
@@ -101,6 +101,13 @@
         <bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-admin-module/${project.version}</bundle>
     </feature>
 
+    <feature name="spatial-geocoding-plugin" install="manual" version="${project.version}"
+             description="Plugin that uses GeoCoder to fetch country codes for geolocated metacards">
+        <feature prerequisite="true">spatial-app</feature>
+        <feature prerequisite="true">webservice-gazetteer</feature>
```

Comment:
This shouldn't prereq the `webservice-gazetteer` since it should function equally as well with the `offline-gazetteer`.